queries.sql

```diff
@@ -129,4 +129,70 @@ FROM animals
 JOIN owners ON animals.owner_id = owners.id 
 GROUP BY owners.full_name 
 ORDER BY COUNT(*) DESC 
-LIMIT 1;
\ No newline at end of file
+LIMIT 1;
+
+/*add "join table" for visits queries*/
+
+SELECT a.name 
+FROM animals a 
+JOIN visits v ON a.id = v.animal_id 
+JOIN vets vt ON v.vet_id = vt.id 
+WHERE vt.name = 'William Tatcher' 
+ORDER BY v.visit_date DESC 
+LIMIT 1;
+
+SELECT COUNT(DISTINCT v.animal_id) 
+FROM visits v 
+JOIN vets vt ON v.vet_id = vt.id 
+WHERE vt.name = 'Stephanie Mendez';
+
+SELECT vt.name, s.name AS specialty
+FROM vets vt
+LEFT JOIN specializations sp ON vt.id = sp.vet_id
+LEFT JOIN species s ON s.id = sp.species_id
+ORDER BY vt.name;
+
+SELECT a.name 
+FROM animals a 
+JOIN visits v ON a.id = v.animal_id 
+JOIN vets vt ON v.vet_id = vt.id 
+WHERE vt.name = 'Stephanie Mendez' 
+AND v.visit_date BETWEEN '2020-04-01' AND '2020-08-30';
+
+SELECT a.name AS animal_name, COUNT(*) AS visit_count 
+FROM animals a 
+JOIN visits v ON a.id = v.animal_id 
+GROUP BY a.id 
+ORDER BY visit_count DESC 
+LIMIT 1;
+
+SELECT a.name AS animal_name, MIN(v.visit_date) AS first_visit
+FROM animals a
+INNER JOIN visits v ON a.id = v.animal_id
+INNER JOIN vets vt ON vt.id = v.vet_id
+WHERE vt.name = 'Maisy Smith'
+GROUP BY a.name;
```

Comment:
- [ ] @Leeoasis great work on your queries however this query didn't show Maisy Smith's first visit but showed all the visits   without any order kindly consider ordering the result of your query and  limiting the result to 1 using ` LIMIT 1;`, Don't forget to update the query result screenshot